src/Prima/Pyxis/Form/RadioButton.elm

```diff
@@ -231,6 +249,17 @@ hasSubtitleAttribute choice =
         Attrs.class ""
 
 
+{-| Internal. Checks RadioButton has subtitle
+-}
+hasDisabledAttribute : Bool -> Html.Attribute msg
+hasDisabledAttribute disabled =
+    if disabled then
+        Attrs.class "is-disabled"
+
+    else
+        Attrs.class ""
+
+
```

Comment:
Questa funzione dovrebbe confluire in un `Maybe.map` alla riga 320.

Reply:
options.disabled è un booleano non Maybe (inizializzato a False) quindi non ha senso che sia dentro un Maybe.map